numpy/fft/fftpack_litemodule.c

```diff
@@ -166,6 +166,10 @@ fftpack_rfftf(PyObject *NPY_UNUSED(self), PyObject *args)
     PyArray_DIMS(data)[PyArray_NDIM(data) - 1] = npts/2 + 1;
     ret = (PyArrayObject *)PyArray_Zeros(PyArray_NDIM(data),
             PyArray_DIMS(data), PyArray_DescrFromType(NPY_CDOUBLE), 0);
+    if (ret == NULL) {
+        PyErr_NoMemory();
```

Comment:
I have wrongly advised you here, the exception is already raised by PyArray_Zeros so this is not necessary